public/app/plugins/datasource/cloudwatch/components/LogsQueryEditor.tsx

```diff
@@ -36,11 +34,6 @@ export const CloudWatchLogsQueryEditor = memo(function CloudWatchLogsQueryEditor
     };
   }
 
-  const { isSyntaxReady, syntax } = useCloudWatchSyntax(
```

Comment:
We are passing syntax directly to PrismSlate plugin, so this is not needed anymore.